node/src/indexer/fake.rs

```diff
@@ -542,14 +591,15 @@ impl Drop for PausedIndexer {
         self.indexer_suspended.send(false).unwrap();
     }
 }
-
+#[derive(Clone, Copy, PartialEq, PartialOrd, Eq, Ord, From)]
+pub struct Uid(usize);
 /// Runs the fake indexer logic for one node.
 struct FakeIndexerOneNode {
     /// Account under which transactions by this node are originated.
-    account_id: AccountId,
+    uid: Uid,
```

Comment:
The docstring is confusing here now. What are these Uids anyway? Node IDs? 

```suggestion
/// Internal ID used to organize nodes for tests
uid: Uid,
```